app/org/sagebionetworks/bridge/models/healthdata/HealthDataAttachment.java

```diff
@@ -0,0 +1,30 @@
+package org.sagebionetworks.bridge.models.healthdata;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import org.sagebionetworks.bridge.dynamodb.DynamoHealthDataAttachment;
+import org.sagebionetworks.bridge.json.BridgeTypeName;
+
+/**
+ * Metadata record about health data attachments. This is used to keep track of attachments in external storage
+ * (generally S3) and the actual health data record.
+ */
+@BridgeTypeName("HealthDataAttachment")
+@JsonDeserialize(as = DynamoHealthDataAttachment.class)
+public interface HealthDataAttachment {
+    /**
+     * Attachment ID. This is used as both the unique attachment ID and the S3 key in the attachments bucket to
+     * guarantee filename uniqueness in S3.
+     */
+    String getId();
+
+    /** Record ID of the health data record this file is attached to. */
+    String getRecordId();
```

Comment:
Not sure what you mean here.

Do you mean split the record so that each record has only one attachment? A multi-attachment record represents multiple recordings of a single event. This is probably relevant to the researchers. Breaking the record up into multiple records means we lose that information.

Alternatively, do you mean store attachments in the records table alongside records instead of creating a new table? This doesn't solve the problem of multiple writes, and the health data records table has too heavy of a schema for attachments.